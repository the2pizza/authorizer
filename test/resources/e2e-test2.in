{"transaction": {"merchant": "Uber Eats", "amount": 25, "time": "2020-12-01T11:07:00.000Z"}}
{"account": {"active-card": true, "available-limit": 225}}
{"transaction": {"merchant": "Uber Eats", "amount": 25, "time": "2020-12-01T11:07:00.000Z"}}