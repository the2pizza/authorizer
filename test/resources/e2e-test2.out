{"account":{},"violations":["account-not-initialized"]}
{"account":{"active-card":true,"available-limit":225},"violations":[]}
{"account":{"active-card":true,"available-limit":200},"violations":[]}